benchmarks/MOABB/train.py

```diff
@@ -253,174 +387,109 @@ def check_if_best(
 
 
 def run_experiment(hparams, run_opts, datasets):
-    """This function performs a single training (e.g., single cross-validation fold)"""
-    idx_examples = np.arange(datasets["train"].dataset.tensors[0].shape[0])
+    """Run a single experiment with the new data format."""
```

Comment:
"New" data format may not mean anything to someone reading this file. Perhaps you can either explicitly write out the data format, or just state more clearly what the function does.